sde_collections/models/collection.py

```diff
@@ -634,6 +634,22 @@ def apply_all_patterns(self):
         for pattern in self.deltadivisionpatterns.all():
             pattern.apply()
 
+    def count_curated_urls(self):
+        """Return the count of Curated URLs for the collection."""
+        return CuratedUrl.objects.filter(collection=self).count()
+
+    def count_dump_urls(self):
+        """Return the count of all Dump URLs for the collection."""
+        return DumpUrl.objects.filter(collection=self).count()
+
+    def count_delta_urls(self):
+        """Return the count of Delta URLs identified."""
+        return DeltaUrl.objects.filter(collection=self).count()
+
+    def count_marked_for_deletion_urls(self):
+        """Return the count of Delta URLs marked for deletion."""
+        return DeltaUrl.objects.filter(collection=self, to_delete=True).count()
+
```

Comment:
I don't think we need these as model functions. Can we move them to the tasks file?